test/unit/specs/table.spec.js

```diff
@@ -815,10 +815,10 @@ describe('Table', () => {
     });
 
     describe('type', () => {
-      const createTable = function(type) {
+      const createTable = function(type, tableProps) {
         return createVue({
           template: `
-            <el-table :data="testData" @selection-change="change">
+            <el-table :data="testData" @selection-change="change" ${tableProps || ''}>
```

Comment:
Is these additions necessary?

Reply:
I was thinking of adding unit tests to cover this prop, but wasn't sure if they should be added to `describe('attributes'` or `describe('type'`.

I've removed the changes for now, but can add them in a new PR if required.